examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>'..besides, our pipeline consists not only of model training; some time is taken by CPU preprocessing like <code>resize</code>  and <code>to_array</code>'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>